### PR TITLE
Stores

### PR DESCRIPTION
**Pull request by mbsyaswanth**

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const usersUrl = 'https://api.github.com/users';
```
Comment (0xc0d3r) [inline src/constants.js]: Which constants do you mean? Change the filename appropriately

File: src/models/Repo/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Repo {
```
Comment (0xc0d3r) [inline src/models/Repo/index.js]: move this folder to `stores/models/`

File: src/stores/UserStore/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import {observable, action} from 'mobx';
+import User from '../../models/User';
+import {apiStatus} from '../../constants';
+import UserServices from '../../services/UserServices/index.api';
```
Comment (0xc0d3r) [inline src/stores/UserStore/index.js]: you should not import this. Accept it as a constructor argument

File: src/stores/UserStore/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+class UserStore {
+  @observable users = [];
+  @observable apiStatus = '';
+  @action.bound setUsers(userData) {
+    this.users = userData.map(user => {
+      new User(user.login, user.avatar_url, user.repos_url);
+    });
+  }
+  @action.bound async getUsers() {
+    this.apiStatus = apiStatus.loading;
```
Comment (0xc0d3r) [inline src/stores/UserStore/index.js:26]: write a method to update the apiStatus

**Review by 0xc0d3r — CHANGES_REQUESTED** [review]

File: src/constants.js (hunk 1 above)
Comment (mbsyaswanth) [reply src/constants.js]: constants
  -ApiStatus
  -Url

**Review by 0xc0d3r — APPROVED** [review]

**Pull request closed without merging**